libs/application/core/src/lib/messages.ts

```diff
@@ -343,6 +343,21 @@ export const coreDefaultFieldMessages = defineMessages({
     defaultMessage: 'Hlaða niður skjali',
     description: 'Button label to download file',
   },
+  defaultQuantityTitle: {
+    id: 'application.system:core.default.pdfLinkButtonField.defaultQuantityTitle',
+    defaultMessage: 'Fjöldi',
+    description: 'Quantity label for payment overview field',
+  },
+  defaultUnitPriceTitle: {
+    id: 'application.system:core.default.pdfLinkButtonField.defaultUnitPriceTitle',
+    defaultMessage: 'Verð',
```

Comment:
Væri ekki betra að kalla þetta "Verð" -> "Ein. verð" og "Samtals verð" -> "Fjárhæð" svo það sé enginn ruglingur?

(er bara að miða við orðin notuð í reikning frá HMS)